cmds/ifdtool/dat.go

```diff
@@ -0,0 +1,127 @@
+/*
+ * ifdtool - dump Intel Firmware Descriptor information
+ *
+ * Copyright (C) 2011 The ChromiumOS Authors.  All rights reserved.
+ *
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation; version 2 of the License.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ */
+package main
+
+import (
+	"fmt"
+	"io"
+)
+
+const (
+	v1 = 1
+	v2
+	linelen = 80
+	magic   = 0x0ff0a55a
+)
+
+const (
+	f20MHZ = 0
+	f33MHZ
+	f48MHZ
+	f50_30MHZ = 4
+	f17MHZ    = 6
+)
+
+const (
+	s512KB = 0
```

Comment:
You have to write `iota` here instead of 0, otherwise all the values will equal 0. By default, it doesn't work like c enums.